docs/integration.rst

```diff
@@ -133,6 +133,11 @@ Follow the steps below to enable Azure Blob Storage logging.
         LOGGING_CONFIG = ...
 
 
+
```

Comment:
Please trim these newlines.

Reply:
@Fokko All done. However, builds are failing with this error:
Errors were encountered while processing:
 oracle-java8-installer
W: --force-yes is deprecated, use one of the options starting with --allow instead.
E: Sub-process /usr/bin/dpkg returned an error code (1)